datahub/metadata/test/test_views.py

```diff
@@ -236,7 +236,7 @@ def test_team_view(metadata_client):
 
 
 class TestServiceView:
-    """Tests for the /metadata/service/ view."""
+    """Tests for the /v4/metadata/service/ view."""
```

Comment:
Does the trailing slash need removing too?

```suggestion
    """Tests for the /v4/metadata/service view."""
```